deploy/charts/rook-ceph/values-external.yaml

```diff
@@ -0,0 +1,54 @@
+# All values below are taken from the CephCluster CRD
```

Comment:
Have you tested this helm example values? From #12066 not sure it's working yet.

Reply:
Not tested yet,

But seems like in the setup networking is not working...

Reply:
Even testing in a single cluster should be possible similar to the CI where the provider is in one namespace and the consumer in another namespace.